homeassistant/components/workday/binary_sensor.py

```diff
@@ -60,7 +60,7 @@ def valid_country(value: Any) -> str:
 
 PLATFORM_SCHEMA = PARENT_PLATFORM_SCHEMA.extend(
     {
-        vol.Required(CONF_COUNTRY): valid_country,
+        vol.Optional(CONF_COUNTRY): valid_country,
```

Comment:
Under ADR-0007: https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md you are not allowed to change the platform schema.

I think this also falls under ADR-0010: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md, and you will need to first migrate to a config flow.